components/camel-casper/pom.xml

```diff
@@ -0,0 +1,290 @@
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.apache.camel</groupId>
+		<artifactId>components</artifactId>
+		<version>3.20.0-SNAPSHOT</version>
+	</parent>
+
+
+	<artifactId>camel-casper</artifactId>
+	<packaging>jar</packaging>
+
+	<name>Camel :: Casper</name>
+	<description>Camel Casper blockchain component</description>
+	<url>https://casperlabs.io/</url>
+
+
+	<properties>
+		<compiler.source>1.8</compiler.source>
+		<compiler.target>1.8</compiler.target>
+		<maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+		<maven-resources-plugin.version>3.2.0</maven-resources-plugin.version>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<log4j2.version>2.13.3</log4j2.version>
+		<camel.version>3.14.2</camel.version>
+		<casper-java-sdk.version>0.3.0</casper-java-sdk.version>
+		<junit.version>4.13.2</junit.version>
+		<launchdarkly.version>2.3.2</launchdarkly.version>
+		<commons-lang3.version>3.11</commons-lang3.version>
+		<casper-sdk.version>0.1.0</casper-sdk.version>
+		<commons-cli.version>1.4</commons-cli.version>
+		<commons-validator.version>1.7</commons-validator.version>
+		<junit-jupiter.version>5.8.0</junit-jupiter.version>
+		<mockito-core.version>3.12.2</mockito-core.version>
+		<mockito-junit-jupiter.version>3.12.2</mockito-junit-jupiter.version>
+		<springframework.version>2.1.2.RELEASE</springframework.version>
+		<fasterxml.jackson.version>2.14.0-rc1</fasterxml.jackson.version>
+		<byte-buddy.version>1.9.3</byte-buddy.version>
+		<objenesis.version>2.6</objenesis.version>
+		<tomcat.version>9.0.54</tomcat.version>
+	</properties>
+
+
+	<dependencies>
+
+
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>${commons-lang3.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>commons-cli</groupId>
+			<artifactId>commons-cli</artifactId>
+			<version>${commons-cli.version}</version>
+		</dependency>
+
+	
+		<dependency>
+			<groupId>org.apache.camel</groupId>
+			<artifactId>camel-jackson</artifactId>
+		
+
+		</dependency>
+		<dependency>
+			<groupId>org.apache.camel</groupId>
+			<artifactId>camel-core</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.camel</groupId>
+			<artifactId>camel-main</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.apache.camel</groupId>
+			<artifactId>camel-jsonpath</artifactId>
+		</dependency>
+
+
+		<dependency>
+			<groupId>org.json</groupId>
+			<artifactId>json</artifactId>
+			<version>20211205</version>
```

Comment:
This kind of lib is using the json license, which is not compatible with Apache license. It should be removed and something else used.